### PR TITLE
added conferences in DE, AT, CH

### PR DESCRIPTION
**Pull request by eumiro**
add Swiss Python Summit, PyDays Vienna, GeoPython and PyCon DE in 2018

### CODE REVIEW

File: 2018.csv [hunk 1]
```diff
@@ -24,4 +27,5 @@ EuroSciPy,2018-08-28,2018-09-01,"Trento, Italy",,,https://www.euroscipy.org/2018
 PyGotham,2018-10-05,2018-10-06,"New York, NY, USA",,,https://2018.pygotham.org,,https://2018.pygotham.org/sponsors/prospectus/
 DjangoCon US,2018-10-14,2018-10-19,"San Diego, CA, USA",,,https://2018.djangocon.us,,
 PythonBrasil,2018-10-17,2018-10-22,"Natal, Rio Grande Do Norte, Brazil",,2018-06-30,http://2018.pythonbrasil.org.br,https://speakerfight.com/events/python-brasil-2018-palestras/,http://2018.pythonbrasil.org.br/#sponsors
+PyCon DE,2018-10-24,2018-10-26,"Karlsruhe, Germany",,,http://pycon.de,,http://pycon.de/sponsoring/
```
Comment (hroncok) [inline 2018.csv]: Maybe use https://de.pycon.org/ ? Also, it says October 24–27

Comment (eumiro) [reply 2018.csv]: de.pycon.org is an option, but the dates are really 24–26 Oct, there is a typo both on the web and on Twitter. On 27–28 Oct there are sprints, not really a part of the conference.

Comment (hroncok) [reply 2018.csv]: Whether sprints are or are not part of the conference, that's not a discussion I want to have. I would rather have the data in sync with the website. So please either change the PR or the website, if that's in your power.

**Comment by hroncok** [comment]
In the meantime, please review https://github.com/python-organizers/conferences/pull/30

File: 2018.csv (hunk 1 above)
Comment (hroncok) [reply 2018.csv]: Fixed

**Pull request merged**